sys/kern/dtb.c

```diff
@@ -0,0 +1,34 @@
+#include <sys/klog.h>
+#include <sys/mimiker.h>
+#include <sys/dtb.h>
+#include <sys/fdt.h>
+#include <sys/kmem.h>
+#include <sys/vm.h>
+
+/* root of dtb in va */
+static void *_dtb_root = NULL;
+/* phys addr of dtb_root */
+static paddr_t _dtb_root_pa = 0;
+/* size of dtb */
+static size_t _dtb_size = 0;
+
+void *dtb_root(void) {
+  assert(_dtb_root != NULL);
+  return _dtb_root;
+}
```

Comment:
[nitpicking] I would move the declaration after the `dtb_early_...` definitions to group it by use.

Reply:
done